recipes-kernel/kernel-tests/kernel-tests-files/kernel_dmesg_diff.py

```diff
@@ -51,13 +51,24 @@ def get_device_desc():
     cmdoutput = run_cmd(['fw_printenv', 'DeviceDesc'])
```

Comment:
Please don't merge this Pull Request until I've closed this comment.

Reply:
Added entries for `kernel_version` field to last 2 current and next kernel logs in our database.